elasticsearch/Dockerfile

```diff
@@ -13,6 +15,18 @@ ENV ENABLE_CONFIGURE_S3="false"
 
 ENV TEMPLATE_VERSION=v3.8.2
 
+ENV ELASTIC_CLUSTER="false" \
```

Comment:
The variables that will be used if $ELASTIC_CLUSTER is true must start with "cluster". Example:
- node_master -> cluster_node_master
- node_data -> cluster_node_data

Also, comment on the file to explain it.